client/my-sites/pages/virtual-page/index.tsx

```diff
@@ -124,31 +130,40 @@ const VirtualPage = ( {
 					onClick={ clickPageTitle }
 				>
 					<span>{ title }</span>
-					{ isHomepage && template && (
+					{ isHomepage && (
 						<InfoPopover position="right">
-							{ translate(
-								'The homepage of your site displays the %(title)s template. {{learnMoreLink}}Learn more{{/learnMoreLink}}.',
-								{
-									args: { title: decodeEntities( template.title.rendered || template.slug ) },
-									components: {
-										learnMoreLink: (
-											<ExternalLink
-												href={ localizeUrl(
-													'https://wordpress.com/support/templates/#template-hierarchy'
-												) }
-												target="_blank"
-												rel="noopener noreferrer"
-											/>
-										),
-									},
-								}
-							) }
+							{ ! isAdmin && template
```

Comment:
Shouldn't this be for admin?

```suggestion
							{ isAdmin && template
```

Reply:
Sorry!